### DISCUSSION TITLE
Iniciando

### DISCUSSION THREAD

**Discussion by JosielCastilloV** [Launch-X-Latam/MisionFrontEnd#469] [Team Posts]
Acabo de unirme a todo office, teams, github por problemas con el correo y estoy viendo los videos, pero aun estoy un poco confundido agradeceria si alguien me explicara con calma o cual es la ruta que puedo seguir para poder finalizar esta mision y seguir a la otra y tratar de ponerme al dia

**Comment by Ruben890** [comment]
La ruta que debes seguir será HTML, CSS, Js VueJs y azure para foront-end.

**Comment by JosielCastilloV** [comment]
Muchas gracias por responder, esta ruta se explica en los videos o se desarrolla en ella o hay otro metodo para seguirla o son independientes?

**Comment by Annamalag** [comment]
Chicos alguno me eneseñaria como hiciern la parte practica para tener una
guia

**Comment by Ruben890** [comment]
Por lo que ví son las rutas que te dan. Ahora yo tengo una pregunta cómo puedo ver los vídeos o acceder a ellos y de igual forma como veo los información de lo de backend

**Comment by Ruben890** [comment]
Acabo de unirme para lo que es el backend

**Comment by JosielCastilloV** [comment]
Es un proceso un poco largo debes acceder con el correo que te dieron de innovaccion virtual, dar la alta a una cuenta de github con ese correo despues agregar la cuenta a la comunidad de github de Lauch y por ultimo cuando ya tienes todo poner el usuario de github y pedir que se te agrege al grupo de teams en el cual se encuentran los videos y todo el contenido

**Comment by Ruben890** [comment]
Y cómo oído ya estoy en el Microsoft team pero no me an agregado a un grupo lo escribo en el chat directamente

**Comment by Ruben890** [comment]
Hago

**Comment by JosielCastilloV** [comment]
El ultimo paso es que agregen al grupo de teams si ya tienes tu correo de innovaccion y cuenta de github con ese correo solo debes ir al siguiente enlace https://forms.office.com/pages/responsepage.aspx?id=BUvlSn63JEKu8YZhQi4IFqqReLKxwkROhdw1fyoxsStUNUZSMjhFUjRPNk03MUgxNFRMSklLTFJBVS4u que esta en el onboard es el ultimo paso

**Comment by Ruben890** [comment]
Gracias

**Comment by Rarriaza-sv** [comment]
https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/40

**Comment by santiudev** [comment]
se puede seguir la mision backend sin realizar el frontend?

**Comment by CarlNauj** [comment]
Buenas noches Estimado:

                Por lo visto en la clase y por correo, las prácticas el 18 de noviembre es la fecha límite y en cuanto al backend uno se puede inscribir, revise su correo.

                Esperando haberle ayudado.

              Saluda atentamente a usted.

JUAN CARLOS CARREÑO POBLETE.

De: Santiago Urban ***@***.***>
Enviado el: sábado, 5 de noviembre de 2022 22:45
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Iniciando (#41)

se puede seguir la mision backend sin realizar el frontend?

**Comment by Rarriaza-sv** [comment]
dejen de hacer spam a los demas , lean antes de escribir
https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/40

**Comment by JoseEduardoVelazquezCN** [comment]
Desactiva las notificaciones o eso habían dicho si te incomoda jajajaj

**Comment by JoseEduardoVelazquezCN** [comment]
Que yo no he preguntado nada solo te estoy diciendo si te molesta las notificaciones y eso dijeron desactiva las notificaciones y listo.... Recuerda que muchos comienzan de 0 y no están familiarizados con las cosas usadas aquí ✌️👍

**Comment by jtaboracastejon** [comment]
Sigue las grabaciones en Team el Sherpa guia todo lo que tienes que hacer actualmente empezamos con el backend
